### ISSUE TITLE
Crash due to Fatal Exception: NSPhotoLibraryUsageDescription not present

### ISSUE THREAD

**Issue by CurlyHeir** [intercom/intercom-ios#206]
## Version info
  - Intercom for iOS version: 3.0.21
  - iOS version: 9.3.5
  - device: iPad Mini

## Steps to reproduce
I couldn't reproduce it, but it occurred on production, we have following logs

## Logs
Stack:
```
	<MyApp>	
__58-[ICMConversationViewController didSelectUploadAttachment]_block_invoke
3	<MyApp>	
-[ICMComposerView didSelectImageUploadButton:]
```
reason: 
```
Fatal Exception: NSPhotoLibraryUsageDescription not present
Your app will be rejected from the AppStore unless you add NSPhotoLibraryUsageDescription to your Info.plist. More information is available here: https://developers.intercom.com/docs/ios-installation#section-step-2-update-infoplist
```
##Additional informations
I provided `NSPhotoLibraryUsageDescription` in MyApp-info.plist

**Comment by bboyle18** [comment]
Hi @CurlyHeir 👋 

Thanks for reporting this issue and apologies for the delay in getting back to you, the team were off for the Christmas break.
I've tried to recreate the issue you have reported and the only way that I can reproduce it is if I don't provide an `NSPhotoLibraryUsageDescription` as part of my plist.

Would you be able to send a screenshot of your plist so that we can help investigate this further?

Thanks

Brian

**Comment by bboyle18** [comment]
Hi @CurlyHeir 

I just wanted to follow up on this issue with you. Were you able to resolve it or can you provide us with some extra info to help us investigate it further?

Thanks

Brian

**Comment by bboyle18** [comment]
Hi @CurlyHeir 

We've just released version [3.0.23](https://github.com/intercom/intercom-ios/releases/tag/3.0.23) which contains a fix that will prevent this happening.

Thanks

Brian
